reference/api/indexes.md

```diff
@@ -88,10 +121,10 @@ Creating an index is an asynchronous task. [You can read more about asynchronous
 
 ### Body
 
-| Variable         | Description                                                |
-| ---------------- | ---------------------------------------------------------- |
-| **`uid`**        | The index unique identifier (_mandatory_)                  |
-| **`primaryKey`** | The primary key of the documents |
+| Name             | Type            | Default value | Description                                                                              |
+| :--------------- | :-------------- | ------------- | :--------------------------------------------------------------------------------------- |
+| **`uid`** *      | String          | N/A           | [`uid`](/learn/core_concepts/indexes.md#index-uid) of the requested index                |
+| **`primaryKey`** | String / `null` | N/A           | [`primaryKey`](/learn/core_concepts/primary_key.md#primary-field) of the requested index |
```

Comment:
This is how I see the behavior: if the `primaryKey` argument is not given, it is treated as if its value is `null`.

```suggestion
| **`primaryKey`** | String / `null` | `null`           | [`primaryKey`](/learn/core_concepts/primary_key.md#primary-field) of the requested index |
```